src/main/java/com/project/service/ApplicationServiceImpl.java

```diff
@@ -41,14 +40,14 @@ public Application addApplication(Application application) throws UserException
 
     @Override
     public void deleteApplication(long id) throws ApplicationNotFoundException {
-        if(!applicationRepository.findById(id).isPresent())
+        if (!applicationRepository.findById(id).isPresent())
             throw new ApplicationNotFoundException(id);
         applicationRepository.deleteById(id);
     }
 
     @Override
     public Application updateApplication(Application application, long id) throws ApplicationNotFoundException {
-        if(!applicationRepository.findById(id).isPresent())
+        if (!applicationRepository.findById(id).isPresent())
```

Comment:
`if` statements should be opened and closed with curly braces `{}` even if they have only one line of code.